redash/utils/__init__.py

```diff
@@ -70,7 +70,13 @@ def generate_token(length):
 
 class JSONEncoder(simplejson.JSONEncoder):
     """Adapter for `simplejson.dumps`."""
-
+    
+    def __init__(self, *args, **kwargs):
+        super().__init__()
+        # Float value nan or inf in Python should be render to None or null in json.
+        # Using ignore_nan = False will make Python render nan as NaN, leading to parse error in front-end
+        self.ignore_nan = True
```

Comment:
Let's make it possible to override + consistent with other options by moving into `json_dumps`:

https://github.com/getredash/redash/blob/37a6e70a4ae06923e156cbb0babf6afae9d933d0/redash/utils/__init__.py#L118-L123

You will just need to add `kwargs.setdefault('ignore_nan', True)` there.

Reply:
Thanks for the review. I have re-implemented as suggestion